demo/src/app/home/home.component.ts

```diff
@@ -44,6 +43,14 @@ export class HomeComponent implements OnInit {
         Validators.email
       ]
     }),
+    new SelectField({
```

Comment:
A read-only SelectField doesn't make much sense. But ok, we can review that later